posts/exploring-iot-infrastructure-support-in-aws-part1.md

```diff
@@ -0,0 +1,188 @@
+[Greengrass](https://aws.amazon.com/documentation/greengrass/) | Works as an extension of IoT core and runs on the devices providing local compute, message, sync and ML inference. | Raspberry Pi, EC2, Texas Instruments (x86/Arm >1GHz, 128MB+ of RAM).| A controller and SDK provided with more compute power with tacked on Lambda and group management, OTA and management device can be anything really. Ubuntu, Raspberry Pi, EC2. Flexible Lambda functionality and ML inference.
+[FreeRTOS](https://aws.amazon.com/documentation/freertos/) | Extension of the standard FreeRTOS. This is the local OS for nodes in our IoT network for devices with low compute power.| Espressif, Microchip, NXP, STM and Texas instruments < 128MB of RAM. | FreeRTOS with additional libs attached to carry out AWS service calls. A low level OS.
+[IoT 1-Click](https://aws.amazon.com/documentation/iot-1-click/) |  Simple service invokes Lambda functions with out of the box device support. | The Enterprise, AT&T or Soracom LTE-M [buttons](https://aws.amazon.com/iot-1-click/devices/). | Button invokes Lambda function. Not very powerful or useful compared to other offerings (Only US).
+[IoT Analytics](https://aws.amazon.com/documentation/iotanalytics/) |  Managed service to collect and query data from IoT devices with Jupyter notebook and Amazon quicksight integration. | - | Have not explored yet (not available in Sydney).
+[IoT Device Management](https://aws.amazon.com/documentation/iot-device-management/) |  Part of the AWS IoT Core feature for onboarding IoT devices, fleet management, serial number management and security policies. | - | Commercial management of devices used in a production environment (larger deployments).
+
+# Current development options (Software, SDKs)
+Before deploying we need to understand the AWS infrastructure and potential options. As a new user in of AWS IoT you might find options confusing and documentation split across many areas. There are many examples for setting up each part of the IoT offerings but little information on actual implementation and integration of features.
+
+## For general devices and clients interacting with IoT devices, infrastructure includes the following:
+
+- [AWS IoT SDK](https://aws.amazon.com/iot/sdk/) on devices
+  - This includes the standard Java, Python, NodeJs SDKs that can access most AWS services including IoT Core and Greengrass
+- [Greengrass Software](https://docs.aws.amazon.com/greengrass/latest/developerguide/gg-config.html)
+  - Standard Greengrass Software (only supported on some devices) runs the greengrass daemon
+  - Will need to use other AWS SDKs if devices are not supported
+
+## For IoT devices at the edge interfacing with sensors and lower level hardware. These require upload and download of sensor data and config including OTA, Sync, etc
```

Comment:
I think this title is too long. You might consider shrinking it, and putting in a small description.